userguide/tutorials/overdue.adoc

```diff
@@ -512,6 +512,139 @@ Even if the payment retry schedule is not configured explicitly, there is a defa
 
+== More about overdue.xml
+
+Earlier, we have taken a look at a basic overdue configuration file. This section provides more details about the contents of this file.
+
+=== Overdue state Information
+
+At its core, the overdue configuration file contains a list of the various states that a customer's account may go through if the customer does not pay. Each state includes the information listed in the table below.
+
+[cols="a,a,a,a"]
+|===
+|*Information*   |*Type*   |*Description* |*Example*  
+//----------------------
+|name   |Attribute on `state` tag   |Name given to the state | 
+[source, xml]
+<state name="WARNING">  
+|condition     |Tag within `state` |Specifies the condition to be evaluated. The account will enter this overdue state only if the condition evaluates to true.  | See <<Overdue Conditions>>
+|externalMessage   |Tag within `state`    |Optional message which can be retrieved by plugins listening to overdue events, and displayed to the user. Any String value can be specified  |
+[source, xml]
+<externalMessage>
+Reached WARNING
+</externalMessage>
+|subscriptionCancellationPolicy   |Tag within `state`|Specifies whether or not to cancel the subscriptions associated with the account (and the cancellation policy that should be used) when this overdue state is reached. Possible values are `NONE`, `IMMEDIATE`, `END_OF_TERM`)     |
+[source, xml]
+<subscriptionCancellationPolicy>
+END_OF_TERM
+</subscriptionCancellationPolicy>`
+|blockChanges   |Tag within `state`|Specifies whether the customer is allowed to make any plan change on the subscriptions when this overdue state is reached. A true/false value needs to be specified. |
+[source, xml]
+<blockChanges>
+true
+</blockChanges>
+|disableEntitlementAndChangesBlocked  |Tag within `state`|Specifies whether the subscriptions should be paused (which means entitlement and billing will be disabled) when this overdue state is reached. A true/false value needs to be specified.    |
+[source, xml]
+<disableEntitlementAndChangesBlocked>
+true
+</disableEntitlementAndChangesBlocked>
+|autoReevaluationInterval   |Tag within `state`   |Specifies the duration after which the overdue state should be reevaluated. The duration needs to be specified with a `unit` (one of `DAYS`, `WEEKS`, `MONTHS`, `YEARS`, `UNLIMITED`) and an integer value. |  [source, xml]
+                <autoReevaluationInterval>
+                 <unit>DAYS</unit>
+                 <number>4</number>
+                </autoReevaluationInterval>
+|isClearState   |Tag within `state`   |Specifies whether the current state is a clear state. A true/false value needs to be specified. |
+[source, xml]
+ <isClearState>true</isClearState>
+|enterStateEmailNotification   |Tag within `state`   |Deprecated|
+NA
```

Comment:
Is this correct? I assumed that this was deprecated based on [DefaultOverdueState#Line76](https://github.com/killbill/killbill/blob/2b80b0445c7baf1f613425bb236a8cb36f1f377a/overdue/src/main/java/org/killbill/billing/overdue/config/DefaultOverdueState.java#L76)

Reply:
Correct - maybe we don't even mention it?

Reply:
Fixed